testing/kfctl/kfctl_go_test.py

```diff
@@ -95,14 +83,17 @@ def test_build_kfctl_go(app_path, project, use_basic_auth, use_istio, config_pat
     # Owned by project kubeflow-ci-deployment.
     os.environ["CLIENT_SECRET"] = "CJ4qVPLTi0j0GJMkONj7Quwt"
     os.environ["CLIENT_ID"] = (
-      "29647740582-7meo6c7a9a76jvg54j0g2lv8lrsb4l8g"
-      ".apps.googleusercontent.com")
+        "29647740582-7meo6c7a9a76jvg54j0g2lv8lrsb4l8g"
+        ".apps.googleusercontent.com")
 
   if use_istio:
     init_args.append("--use_istio")
   else:
     init_args.append("--use_istio=false")
 
+# Its not clear why this code is needed.
+# Will delete if approved.
+def dummy_version_repo_master():
```

Comment:
I think this is obsolete. I think it was used in previous iterations when we were using a kfctl flag to specify the version of the repo to use.